### DISCUSSION TITLE
Webstorm support

### DISCUSSION THREAD

**Discussion by de-don** [maizzle/framework#1311] [General]
Are you planning to develop a plugin for WebStorm, or maybe there is some kind of workaround to work with this IDE?

**Comment by cossssmin** [comment]
If you're referring to Tailwind, it should work out of the box?

https://www.jetbrains.com/help/webstorm/tailwind-css.html

**Reply by de-don** [reply]
Tailwind works great, it's referring to "components" in maizzle.

Code navigation (to components), validation of `{{ var }}` variables usage, yml definition of variables in templates, and etc doesn't work as expected in Webstorm.

It still compiles and etc, but dev experience is not good :D

**Reply by cossssmin** [reply]
That would require a Webstorm plugin which wouldn't be a trivial project - we currently have no plans on developing one but welcome any contributions 👍
